secure-storage/secure-storage-impl/build.gradle

```
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'com.squareup.anvil'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+dependencies {
+    implementation project(path: ':appbuildconfig-api')
+    implementation project(path: ':di')
+    implementation project(path: ':secure-storage-api')
+    api project(path: ':secure-storage-store')
```

Although we've done this in the past, this is strictly not correct according to [this](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation). 
I was planning to tackle all these in a clean up task later on but just in case you want to fix it now.

The fix would be to turn this into an `implementation` and add the `implementation project(path: ':secure-storage-store')` into the `:app` module.
```suggestion
    implementation project(path: ':secure-storage-store')
```

---

Right. I would assume  that would mean that we would inject `:secure-storage-store` into the app's build.gradle instead?